### DISCUSSION TITLE
Glassit

### DISCUSSION THREAD

**Discussion by Filipriec** [VSCodium/vscodium#1592] [Q&A]
After adding Glassit i cant see any transparency, my vscode works fine. Using linux mint. Is it only me, am i doing something wrong? PS: picom havent achieved it also.

**Comment by R-J-N-1810** [comment]
Hey I am facing the same problem now.
Did you resolve it?

**Reply by Filipriec** [reply]
Transparency on the text editor is like the worst thing ever. It looks good and cool, but after some months I made it nontransparent. My current IDE is emacs.
The way I fixed it if I remember correctly, was setting it up in picom and making the whole window transparent on the OS level.
